src/lib/bundle.js

```diff
@@ -0,0 +1,37 @@
+import FTLRuntimeParser from './format/ftl/entries/parser';
+import { format } from './resolver';
+
+export class Bundle {
```

Comment:
I named this `Bundle` as in the MessageBundle, but perhaps this could called Context or MessageContext.  If we change the name then Context could be called something else to avoid confusion, but I'm out of ideas.